### PR TITLE
Added PWA screenshots and shortcut support

### PR DESCRIPTION
**Pull request by JelleBootsma**
Added support for PWA screenshots as described in [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/Manifest/screenshots) and PWA shortcuts as described in [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/Manifest/shortcuts)

### CODE REVIEW

**Comment by JelleBootsma** [comment]
@silviolleite Can this PR be reviewed / merged? I have already added the unit tests and updated the README

**Comment by pdelboca** [comment]
This PR would be very interesting to merge! Does it provide support for several screenshots?

**Comment by JelleBootsma** [comment]
> This PR would be very interesting to merge! Does it provide support for several screenshots?

Yeah, it supports adding both multiple screenshots and multiple shortcuts

File: pwa/app_settings.py [hunk 1]
```diff
@@ -107,3 +107,17 @@
...
 PWA_APP_DIR = getattr(settings, 'PWA_APP_DIR', 'auto')
 PWA_APP_LANG = getattr(settings, 'PWA_APP_LANG', 'en-US')
+PWA_APP_SCREENSHOTS = getattr(settings, 'PWA_APP_SCREENSHOTS', [
+    {
+      'src': '/static/images/icons/splash-750x1334.png',
+      'sizes': '750x1334',
+      "type": "image/png"
+    }
+  ])
+PWA_APP_SHORTCUTS = getattr(settings, "PWA_APP_SHORTCUTS", [
```
Comment (hartungstenio) [inline pwa/app_settings.py:117]: I'm wondering if we should add the Home as a default shortcut or leave it empty 🤔

Comment (JelleBootsma) [reply pwa/app_settings.py:117]: I think it wouldn't add much, and might be counter intuitive, as shortcuts is an optional field, so if nothing is changed from the default settings, I would expect it to be empty.
 
If a user wants a shortcut to Hom, then they can always add it.

**Pull request merged**